astroid/inference.py

```diff
@@ -802,10 +802,7 @@ def infer_binop(self, context=None):
     "in": lambda a, b: a in b,
     "not in": lambda a, b: a not in b,
 }
-UNINFERABLE_OPS = {
-    "is",
-    "is not",
-}
+UNINFERABLE_OPS = frozenset(("is", "is not"))
```

Comment:
I'm not sold on this. There's no measurable performance improvement, readability is arguably worse off. In my opinion, use of frozenset should be limited to the cases where the intended use of the value is a dictionary key.

Reply:
Using `frozenset` instead of `set` was suggested to me a while ago for a similar case in `pylint`.
After your comment, I did look up the differences and you do have a point here. I'll revert that change.